courses/machine_learning/deepdive2/structured/solutions/3_automl_tables_babyweight.ipynb

```diff
@@ -0,0 +1,487 @@
+{
```

Comment:
<p>Two "just"s in the second to last sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Whoops, should be Since we just, thanks.</p>